node.go

```diff
@@ -168,19 +168,19 @@ func (nc *nodeCollector) collectNode(ch chan<- prometheus.Metric, n api.Node) {
 	addResource(descNodeStatusAllocateablePods, n.Status.Allocatable, api.ResourcePods)
 }
 
-// nodeConditionMetrics generates one metric for each possible node condition status.
-func nodeConditionMetrics(ch chan<- prometheus.Metric, desc *prometheus.Desc, name string, cs api.ConditionStatus) {
+// addConditionMetrics generates one metric for each possible node condition status.
```

Comment:
Doc comment that this requires the condition metric description to have "condition" as the last label.
That kind of collides with the suggestion to have label names alphabetically sorted. But that's fine for now.